### PR TITLE
Fix labels in Aircraft dataset.

### LINKED ISSUE #402: Aircraft dataset num classes = 51?
Hey everyone,

I've been trying to use the aircraft dataset and somehow I can only find 51 classes in it with the preset splits. If I remove the 'if' clause that checks for adherence to the splits I get 70 classes. 

What's going on? Any ideas?

```python
import learn2learn as l2l
from tqdm.auto import tqdm

data = l2l.vision.datasets.FGVCAircraft(
    root="data/",
    mode="all",
    download=True,
    bounding_box_crop=False,
)

label_set = set()

with tqdm(total=len(data)) as pbar:
    for item in data:
        label_set.add(item[1])
        pbar.update(1)
        pbar.set_description(f"Found {len(label_set)} labels")
```

If one runs the above, it returns "Found 51 labels"

### PR DESCRIPTION
**Pull request by seba-1511**
### Description

Fixes #402 

Changes the labels in FGVC Aircrafts to be variants instead of families, as in Meta-Dataset.

### Contribution Checklist

If your contribution modifies code in the core library (not docs, tests, or examples), please fill the following checklist.

- [x] My contribution is listed in CHANGELOG.md with attribution.
- [x] My contribution modifies code in the main library.
- [x] My modifications are tested.
- [ ] My modifications are documented.

### CODE REVIEW

**Comment by seba-1511** [comment]
@AntreasAntoniou it turns out the fix was pretty simple: just build the image_labels.pkl archive using the files with the aircraft variants instead of the families. When rerunning your script (requires re-downloading the data) I get 70 train, 15 validation, and 15 test classes.

Do you want to take a look before I merge?

**Comment by AntreasAntoniou** [comment]
I am having a look right now.

Thanks for tackling this so quickly. I was intending to do that today, but it seems you beat me to it. I did an all nighter on Sunday so I needed to turn in early yesterday, or I would have been able to deliver yesterday!

File: learn2learn/vision/datasets/fgvc_aircraft.py [hunk 1]
```diff
@@ -115,7 +115,8 @@ def __init__(
         self.transform = transform
         self.target_transform = target_transform
         self.bounding_box_crop = bounding_box_crop
-        self._bookkeeping_path = os.path.join(self.root, 'fgvc-aircraft-' + mode + '-bookkeeping.pkl')
+        self._bookkeeping_path = os.path.join(
+            self.root, 'fgvc-aircraft-' + mode + '-bookkeeping.pkl')
```
Comment (AntreasAntoniou) [inline learn2learn/vision/datasets/fgvc_aircraft.py:119]: I recommend using pathlib.Path for general file system operations. It's a lot more versatile and intuitive.

**Review by AntreasAntoniou — APPROVED** [review]

**Comment by AntreasAntoniou** [comment]
Seems good to go! The only thing that I would recommend is using pathlib.Path for filesystem operations. It's a lot more fun to use, and generally a lot more resilient (gets ahead of problems).

**Comment by seba-1511** [comment]
Thanks for the quick review! Agreed re `pathlib` but since we've been using `os` throughout the lib I'll keep it that way.

**Comment by AntreasAntoniou** [comment]
Fair enough :)

**Pull request merged**